Documentation/gettingstarted/cli-download.rst

```diff
@@ -12,14 +12,23 @@ various features (e.g. clustermesh, Hubble).
       sudo tar xzvfC cilium-linux-amd64.tar.gz /usr/local/bin
       rm cilium-linux-amd64.tar.gz{,.sha256sum}
 
-  .. group-tab:: macOS
+  .. group-tab:: macOS (Intel)
 
     .. code-block:: shell-session
 
       curl -L --remote-name-all https://github.com/cilium/cilium-cli/releases/latest/download/cilium-darwin-amd64.tar.gz{,.sha256sum}
       shasum -a 256 -c cilium-darwin-amd64.tar.gz.sha256sum
       sudo tar xzvfC cilium-darwin-amd64.tar.gz /usr/local/bin
       rm cilium-darwin-amd64.tar.gz{,.sha256sum}
+      
+   .. group-tab:: macOS (Apple Silicon (ARM))
```

Comment:
```suggestion
   .. group-tab:: macOS (Apple Silicon - ARM)
```

Nit: Easier to read than embedded parenthesis?

Alternatively, Documentation/gettingstarted/istio.rst uses `OSX` and `OSX (Apple Silicon)`.